msg/NamedPose.msg

```diff
@@ -0,0 +1,9 @@
+# A named pose defined in the simulation for certain purposes such as spawning.
+
+string name                               # Unique name.
+string description                        # Description for the user, e.g. "near the charging station".
+string[] tags                             # Optional tags which can be used to determine the named pose
+                                          # purpose, for example: "spawn", "parking", "navigation_goal",
+                                          # as well as fitting entity types e.g. "drone", "turtlebot3".
+geometry_msgs/Pose pose                   # Pose relative to world, which can be used with SpawnEntity.srv.
+PoseBounds bounds                         # Pose area bounds, which are empty by default, meaning pose is unbounded.
```

Comment:
This feel too specific and probably shouldn't be in the definition of a 'pose'.

Reply:
I believe that the opposite is true. That the hitherto approach to spawn points and any other semantic points in simulators is inadequate, makes it clunky to ensure stability, validate externally etc., and that (optional) bounds will make the situation better.

Reply:
After some deliberation, I decided to go with:
- Generalizing PoseBounds to Bounds. It was confusing and the message is more generic (applies to entities, conditions associated with poses, spawn limits etc).
- Removing pose_bounds field from NamedPoses.
- Adding a new service GetNamedPoseBounds.
- Adding a service to query entity bounds for utility and consistency (GetEntityBounds).